### DISCUSSION TITLE
Tests failing

### DISCUSSION THREAD

**Discussion by Russell500** [gp-seattle/rms-app#125] [Backend Questions] [answered]
![image](https://user-images.githubusercontent.com/87156867/128070234-3930f32d-3268-48b5-81ef-077f77498264.png)
Running into an error where "No Cognito Identity pool provided for unauthenticated access"
and also the code you worked on earlier keeps failing too with a different error

**Comment by yauj** [comment, marked as answer]
Summary of offline convo:
You need to pull in the latest credentials for the integration tests to work. Do this by running `amplify init` again. Will create a task to automatically do this whenever you run build.
